### PR TITLE
feat: remove react-hot-loader

### PR DESCRIPTION
**Pull request by wardpeet**
## Description

Removes react-hot-loader and sets fast-refresh as the only default.

### Documentation

Not that I know of

### CODE REVIEW

File: e2e-tests/development-runtime/package.json [hunk 1]
```diff
@@ -4,18 +4,18 @@
   "version": "1.0.0",
   "author": "Dustin Schau <dustin@gatsbyjs.com>",
   "dependencies": {
-    "gatsby": "^2.27.1",
+    "gatsby": "3.0.0-next.5-dev-1613644438131",
```
Comment (sidharthachatterjee) [inline e2e-tests/development-runtime/package.json:7]: Probably want to clean these up

**Review by sidharthachatterjee — CHANGES_REQUESTED** [review]
You'll want to bump up Gatsby's react devDependencies to a minimum version that supports fast refresh

**Comment by wardpeet** [comment]
> You'll want to bump up Gatsby's react devDependencies to a minimum version that supports fast refresh

That's for another PR

**Comment by sidharthachatterjee** [comment]
@wardpeet Alright

**Review by sidharthachatterjee — APPROVED** [review]

**Pull request merged**